### DISCUSSION TITLE
We need to talk

### DISCUSSION THREAD

**Discussion by Adarsh-jaiss** [collab-community/coding-youtubers#31] [Q&A] [answered]
Are you guys planning to create a website for listing coding-youtubers @KendallDoesCoding @Agrim-Sharma174 @vikasganiga05 

If yes, then I have an  idea💡
What if we conduct a small website making competition for folks and we will select one website and merge it on our codebase as coding youtubers website :) 

This will open up a vast opportunity for folks to work on a similar project and it will give them a little feeling of Hackathon too 😀

And also we will set a deadline of 7 days, to summit there created website by raising an issue named website submission... And they will upload there whole codebase to there repository and they will link that repository while creating that issue. And after that our community moderators will judge and select one website and congratulate them by adding their code in our code base.... And the one who is unelected will be also uploaded at our coding competition repository's codebase by creating folders of each person/team

**Comment by KendallDoesCoding** [comment, marked as answer]
Not as of now, I just wanted to host via GitHub pages so the README.md with the youtubers is visible on the website as well, but we may do your idea in the future!

Tip: no need to ping us all one by one, just ping the team @collab-community/coding-youtubers and you got it!
